stores/moderationSubmissionsStore.ts

```diff
@@ -23,6 +23,11 @@ export const useModerationSubmissionsStore = defineStore(
             submissionsData.value = submissionsSearchResults
         }
 
+        async function getProfessionals() {
```

Comment:
Not needed since we can use the `submissionsData` healthcareProfessionsal field

Reply:
^ could we keep the method, but it just pulls out the professionals from the submission objects?

Reply:
@ermish In my understanding the submissions get fetched when the moderation panel is accessed. So all the data is there. 
This is the data that is been fetched:
```
type Submission {
  id: ID!
  googleMapsUrl: String!
  healthcareProfessionalName: String!
  spokenLanguages: [Locale!]!
  facility: FacilitySubmission
  healthcareProfessionals: [HealthcareProfessionalSubmission!]
  isUnderReview: Boolean!
  isApproved: Boolean!
  isRejected: Boolean!
  createdDate: String!
  updatedDate: String!
  notes: String
}
```

So all we need to do is display the `facility` and `healthcareProfessionals` fields.

Reply:
I kept it but just made a new object with the filtered healthcareprofessionals if we want to use that is any way. I also made a different view. I don't know what kind of link we might want to add. Right now it links you to the Facility Form. But @Anissa3005 How are you going to display the healthcare professional form? Should it be based on a different page? This might be easiest with `editsubmission/healthcare-professional` or something like that